apps/web/app/(app)/environments/[environmentId]/surveys/[surveyId]/(analysis)/summary/components/ShareEmbedSurvey.tsx

```diff
@@ -0,0 +1,96 @@
+import LinkTab from "./shareEmbedTabs/LinkTab";
```

Comment:
since this component is using a useState, please use `"use client"` at the top of this file